notebooks/ncx-equivalency.ipynb

```diff
@@ -0,0 +1,874 @@
+       "41"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "round(joos_2013[101] * 100)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "65592950",
+   "metadata": {},
+   "source": [
+    "#### Before discounting, the ton-year cost of an emission with a 100 year time horizon = 53.07 ton years\n",
+    "\n",
+    "Parisa et al. (2021)\n",
+    "\n",
+    "> A ton-year is defined as one ton of carbon held for a period of one year in any carbon pool. Using\n",
+    "> the Bern model, we determine the number of ton-years removed from the atmosphere as the result of\n",
+    "> one ton “permanently” sequestered into the biosphere. This is the area under the curve in Figure\n",
+    "> 1, which is 53.07 ton-years of impact on the climate system. In our approach, we track the tons\n",
+    "> for 100 years to be consistent with the 100-year GWPs (GWP100) convention\n",
+    "\n",
+    "Jenkins et al. (2021)\n",
+    "\n",
+    "> Whereas the total carbon represented under the curve of Figure 1 is 53.07 ton-years...\n",
+    "\n",
+    "Pg. 19/30 of V0.5 RISE white paper\n",
+    "\n",
+    "> In other words, based on the shape of this curve, 1 ton CO2e is equal to a stream of about 53\n",
+    "> ton-years, delivered over 100 years.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "38ccce05",
+   "metadata": {},
+   "source": [
+    "When we integrate under the joos_2013 curve — as we do in the `tonyear` module calculations — we do\n",
+    "not replicate this claim. However, Parisa et al.'s supplementary materials (page 4, Section I.c.)\n",
+    "discuss performing this calculation via summation of the IRF's timestep values rather than\n",
+    "integration of the IRF curve. Using the discrete summation method on the joos_2013 curve, we can\n",
+    "replicate the 53.07 ton year claim.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "e6cce211",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "52.36"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# integration method\n",
+    "round(np.trapz(joos_2013[:101]), 2)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "94680d5a",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "53.07"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# summation method\n",
+    "round(sum(joos_2013[:101]), 2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "92fe61b6",
+   "metadata": {},
+   "source": [
+    "#### After discounting, the ton-year cost of an emission with a 100 year time horizon = 18.69. With a 1000 year time horizon, the ton-year cost is 19.11.\n",
```

Comment:
I think this heading has a typo, as it refers to 19.11, whereas the output and the quoted text refer to 19.12